### PR TITLE
Replaced portal API for "ADO_Project_AuthZ_Disable_Inherited_Permissions"

### PR DESCRIPTION
**Pull request by v-soukum**
## Description
</br>
Replaced portal API for project control "ADO_Project_AuthZ_Disable_Inherited_Permissions".

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 1]
```diff
@@ -862,71 +862,52 @@ class Project: ADOSVTBase
...
-                        $controlResult.AddMessage([VerificationResult]::Error, "Could not fetch details for repository $($repo.name). Please verify from portal whether inherited permissions is disabled.");
+                    else
+                    {
+                        $passedRepos += $repo.name
                     }
                 }
-                catch {
-                    $controlResult.AddMessage([VerificationResult]::Error, "Could not fetch details for repository $($repo.name). Please verify from portal whether inherited permissions is disabled. $($_).");
+
+                $failedcount = $failedRepos.Count
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:890]: failedCount->failedReposCount

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 2]
```diff
@@ -862,71 +862,52 @@ class Project: ADOSVTBase
...
+                $passedcount = $passedRepos.Count
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:891]: passedcount -> passedReposCount

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 3]
```diff
@@ -862,71 +862,52 @@ class Project: ADOSVTBase
...
+                $passedRepos = $passedRepos | sort-object
+                if($failedcount -gt 0)
+                {
+                    $failedRepos = $failedRepos | sort-object
+                    $controlResult.AddMessage([VerificationResult]::Failed, "Inherited permissions are enabled on the below $($failedcount) repositories:", $failedRepos);
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:896]: What if count is 1? The message will be printed as  below which is gramatically wrong
"Inherited permissions are enabled on the below 1 repositories"
Instead can we add separate message to control result for count?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 4]
```diff
@@ -862,71 +862,52 @@ class Project: ADOSVTBase
...
+                    $controlResult.AddMessage("Inherited permissions are disabled on the below $($passedcount) repositories:", $passedRepos);
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:897]: Same comment applies here

**Review by SaikumarRagala — CHANGES_REQUESTED** [review]

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 (hunk 1 above)
Comment (v-soukum) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1]: changes done

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 (hunk 3 above)
Comment (v-soukum) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1]: changes done

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 (hunk 4 above)
Comment (v-soukum) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1]: changes done

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 (hunk 2 above)
Comment (v-soukum) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1]: changes done

**Review by arvindsingh-cloud — APPROVED** [review]
looks good

**Review by abhaydaga — APPROVED** [review]

**Pull request merged**